docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -118,22 +118,32 @@ tarballs:
 If the system testing is successful, the *preliminary distribution tarballs* used
 in this step become *accepted distribution tarballs*.
 
-## 9. Upload to CPAN
+## 9. Update Zonemaster repository main _README.md_
+
+If needed, update the following sections of the Zonemaster repository main _README.MD_ file:
+
+* Supported operating system versions
+* Supported database engine versions
+* Supported Perl versions
```

Comment:
Updating supported versions for OSes, DB engines and Perl is already addressed in step 3 of this document. Please remove those items here.